main/src/test/java/org/mobilitydata/gtfsvalidator/validator/StopTimeArrivalAndDepartureTimeValidatorTest.java

```diff
@@ -87,34 +86,6 @@ public void departureTimeAfterArrivalTimeShouldNotGenerateNotice() {
     assertThat(noticeContainer.getValidationNotices()).isEmpty();
   }
 
-  @Test
-  public void departureTimeBeforeArrivalTimeShouldGenerateNotice() {
```

Comment:
`departureTimeAfterArrivalTimeShouldNotGenerateNotice` the test just above this one should be removed too.

Reply:
Good catch!